src/js/view/controls/controlbar.js

```diff
@@ -246,6 +269,10 @@ export default class Controlbar {
         _model.change('cues', this.addCues, this);
         _model.change('altText', this.setAltText, this);
         _model.change('customButtons', this.updateButtons, this);
+        _model.change('state', () => {
+            // Check for change of position to counter race condition where state is updated before the current position
+            _model.once('change:position', this.checkDvrLiveEdge, this);
```

Comment:
Do we need to check this every time the position changes? I feel like we can check when the user pauses, plays or seeks or the player stalls. Thoughts?